### ISSUE TITLE
GSConv implementation

### ISSUE THREAD

**Issue by zeobec** [AlanLi1997/slim-neck-by-gsconv#34]
Currently, im trying to implement GSConv module into Yolov10b module.  
conv.py
![image](https://github.com/user-attachments/assets/b8a3cc28-d0c4-4dbb-bdfd-cd424ff4c7fa)

task.py
![image](https://github.com/user-attachments/assets/eaa6707b-f30e-4ae5-867d-46d0c2e88e74)

also imported module in init.py
![image](https://github.com/user-attachments/assets/9b93512f-db2c-47e8-969d-695cce56fb02)

however, i received this error message 
![image](https://github.com/user-attachments/assets/5ed23f5b-86e4-4405-b5da-f0946a92c312)
![image](https://github.com/user-attachments/assets/25e2ce29-6de0-4621-b0fb-1f174d027728)

What's the possible reason that could be the source of the error?

**Comment by AlanLi1997** [comment]
@zeobec You should note the definition of the Conv in YOLOv10. Try to use this GSConv:

      class GSConv(nn.Module):
          # GSConv https://github.com/AlanLi1997/slim-neck-by-gsconv
          def __init__(self, c1, c2, k=1, s=1, g=1, d=1, act=True):
              super().__init__()
              c_ = c2 // 2
              self.cv1 = Conv(c1, c_, k, s, None, g, d, act)
              self.cv2 = Conv(c_, c_, 5, 1, None, c_, d, act)
      
          def forward(self, x):
              x1 = self.cv1(x)
              x2 = torch.cat((x1, self.cv2(x1)), 1)
              # shuffle
              y = x2.reshape(x2.shape[0], 2, x2.shape[1] // 2, x2.shape[2], x2.shape[3])
              y = y.permute(0, 2, 1, 3, 4)
              return y.reshape(y.shape[0], -1, y.shape[3], y.shape[4])

I have not checked the definition of "Conv" in YOLOv10 yet, so please don't hesitate to let me know if you have any further questions.

**Comment by zeobec** [comment]
hi, thankyou for your response. i noticed that you have added new parameter which is d. Based on my understanding, is it true that d is representing dilation? would you mind to explain why you add the new parameter to make the module works?
